app/src/main/java/com/example/concordia_campus_guide/Activities/RoutesActivity.java

```diff
@@ -182,35 +182,34 @@ public void directionsApiCallBack(DirectionsResult result, List<Route> routeOpti
         setRoutesAdapter();
     }
 
-
     private void setRoutesAdapter() {
         // Android adapter for list view
         adapter = new RoutesAdapter(this, R.layout.list_routes, mViewModel.getRouteOptions());
         allRoutes.setAdapter(adapter);
         allRoutes.setOnItemClickListener(new AdapterView.OnItemClickListener() {
             @Override
             public void onItemClick(AdapterView<?> adapterView, View view, int i, long l) {
-                if(carButton.isSelected() || transitButton.isSelected() || walkButton.isSelected()){
+                if (carButton.isSelected() || transitButton.isSelected() || walkButton.isSelected()) {
                     Intent openPaths = new Intent(RoutesActivity.this,
                             PathsActivity.class);
                     DirectionsRoute directionsResult = mViewModel.getDirectionsResult().routes[i];
                     openPaths.putExtra("directionsResult", directionsResult);
+                    openPaths.putExtra("shuttle", false);
                     startActivity(openPaths);
-                }
-                else{
-                    Toast.makeText(getApplicationContext(), "Paths view for Shuttle route has not been implemented yet.", Toast.LENGTH_SHORT).show();
-                    // TODO: Add shuttle route to map #51
-                    // Intent openPaths = new Intent(RoutesActivity.this,
-                    //         PathsActivity.class);
-                    // DirectionsRoute directionsResult = mViewModel.getDirectionsResult().routes[i];
-                    // openPaths.putExtra("directionsResult", directionsResult);
-                    // startActivity(openPaths);
+                } else {
+                    if (mViewModel.getShuttles() == null) {
+                    Toast.makeText(getApplicationContext(), "Paths view for Shuttle route is not available if no shuttles exist.", Toast.LENGTH_SHORT).show();
```

Comment:
Do we need to show a toast for this? Can't we display it on the routesActivity page?

Reply:
if there's no shuttles available the routeOptions will have an element saying "No shuttles available at the time". The toast is just to make sure if someone clicked this option the application will not break